app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -1,5 +1,7 @@
 import * as yup from "yup";
 
+import Avatar from "../../../images/avatar.png";
```

Comment:
https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663146590843649?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

Reply:
This is resolved.
I have moved the image to previously missed assets/images folder.